CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.x.x - 2021-05-21
+
+### Added 
+
+- Import helper filter-object
```

Comment:
O correto seria isso ser um fix:

- Fixed import of `filterObject` helper.

Reply:
Feito, criado mixin filterObjectMixin

Reply:
corrigido